koku/api/utils.py

```diff
@@ -602,3 +603,10 @@ def get_months_in_date_range(
 
     months = dh.list_month_tuples(dt_start, dt_end)
     return [(start, end, dt_invoice_month) for start, end in months]
+
+
+def safe_column_alias(alias: str) -> str:
+    """
+    Sanitize column alias to be SQL-safe (no spaces, quotes, etc.).
+    """
+    return re.sub(r"[^a-zA-Z0-9_]", "_", alias)
```

Comment:
So, I think you are doing this work a bit early. We can change the tag but then it won't match what is stored in the database. I think for now all we want to do is utilize the santized tag correctly.